docs/source/guides/index.rst

```diff
@@ -14,7 +14,7 @@ into this part of the documentation.
 
   batching
   configuration
-  dockerfile_generation
+  containerization
```

Comment:
since we want to eventually support OCI-spec, probably a good idea to move away from just docker :)